### ISSUE TITLE
Small requests / tips with messages and tiers

### ISSUE THREAD

**Issue by zannapzg** [pronze/SBAHypixelify#33]
1.
I saw that in the plugin config you can now enter the name of the minerals (emerald, diamond)
![image](https://user-images.githubusercontent.com/61860483/102950207-701d8f00-44ca-11eb-86e3-0fec29cf39b5.png)

But still not appears :(
I'd like to customize 100% Diamond with Diamante

2.

The scoreboard tiers line is not customizable
![image](https://user-images.githubusercontent.com/61860483/102950586-36995380-44cb-11eb-9e56-2dc617f7a51e.png)

If u can put a section for them..
Obviously, making each event customizable from the generators at the end of the game

3.

The titles appears in chat
![image](https://user-images.githubusercontent.com/61860483/102950735-824bfd00-44cb-11eb-954f-37f18a8603c2.png)

If you can add a section for
  respawn-message
  respawned-message
  
  Because in chat I use a prefix in all messages ("BW>" first) and a lot out of place
  
![2020-12-23_03 03 27](https://user-images.githubusercontent.com/61860483/102950838-bb846d00-44cb-11eb-8e8f-37067757d2ac.png)

In short, I ask you to put everything customizable AHAHAH

**Comment by pronze** [comment]
Done in the latest snapshot

https://ci.screamingsandals.org/job/SBAHypixelify/77/
